core/file/dicom/image.cpp

```diff
@@ -245,8 +245,14 @@ namespace MR {
             }
             return;
           case 0x2001U: // Philips DW encoding info:
-            if (item.element == 0x1003)
-              bvalue = item.get_float (0, bvalue);
+            switch (item.element) {
+              case 0x100:
```

Comment:
```suggestion
              case 0x1003:
```
?

Reply:
Oh, wow.... Thanks for picking that one up! And hooray for code reviews...